http/vibe/http/common.d

```diff
@@ -336,13 +337,222 @@ class HTTPStatusException : Exception {
 	string debugMessage;
 }
 
+final class MultiPartBodyPart
```

Comment:
well the preamble and epilogue are so rarely used that they aren't even documented in the w3c spec, only in some HTTP specs I could find. I think if you write lazily there shouldn't be any support for these